internal/activities/validate_file_formats_test.go

```diff
@@ -24,6 +25,8 @@ func TestValidateFileFormats(t *testing.T) {
 		fs.WithFile("file2.png", pngContent),
 	).Path()
 
+	PREMISFilePath := fs.NewFile(t, "premis.xml", fs.WithContent(premis.EmptyPremis)).Path()
```

Comment:
I think it would good to test the premis.xml file contents in these tests too.

Reply:
Fixed!